docs/edit_first_game_server.md

```diff
@@ -1,11 +1,15 @@
 # Getting Started
-The following guide is for developers without Docker or Kubernetes experience, that want to use the simple-udp example as a starting point for a custom game server. Since this guide is for Google Kubernetes Engine, we welcome a Pull Request to expand this to include Minikube as well.
+The following guide is for developers without Docker or Kubernetes experience, that want to use the simple-udp example as a starting point for a custom game server. This guide addresses Google Kubernetes Engine and Minikube.  We would welcome a Pull Request to expand this to include other platforms as well.
 
 ## Prerequisites
 
 1. Downland and install Golang from https://golang.org/dl/.
 2. Install Docker from https://www.docker.com/get-docker.
-3. Follow the install instructions (if you haven't already) at [Install and configure Agones on Kubernetes](../install/README.md). At least, you should have "Setting up a Google Kubernetes Engine (GKE) cluster", "Enabling creation of RBAC resources" and "Installing Agones" done.
+3. Install Agones on GKE or Minikube.  
+
+To install on GKE, follow the install instructions (if you haven't already) at [Install and configure Agones on Kubernetes](../install/README.md). At least, you should have "Setting up a Google Kubernetes Engine (GKE) cluster", "Enabling creation of RBAC resources" and "Installing Agones" done.
```

Comment:
We can link to the GKE section directly:
https://github.com/GoogleCloudPlatform/agones/blob/master/install/README.md#setting-up-a-google-kubernetes-engine-gke-cluster

WDYT?

Reply:
Yes, that would be more consistent.

Reply:
This change has been made.